src/scripting/cql/context.rs

```diff
@@ -300,9 +298,9 @@ pub struct Context {
     session: Option<Arc<Session>>,
     page_size: u64,
     statements: HashMap<String, Arc<PreparedStatement>>,
-    stats: TryLock<SessionStats>,
+    pub stats: TryLock<SessionStats>,
```

Comment:
It is exposed to rune scripting.
There must be a reason for doing so.
What is it?